src/insns/cjalr_jalr_32bit.adoc

```diff
@@ -24,11 +24,20 @@ Capability Mode Description::
 CJALR allows unconditional, indirect jumps to a target capability. The
 target capability is obtained by unsealing `cs1` and incrementing its address by the
 sign-extended 12-bit immediate, and then setting the least-significant bit of the
-result to zero. The target capability may have
+result to zero. The target address may then have
 xref:section_invalid_addr_conv[xrefstyle=short]
-performed and is then installed in <<pcc>>. The <<pcc>>
-of the next instruction following the jump (<<pcc>> + 4) is sealed and written
-to `cd`.
+performed. The check for length violation is done next using the converted
+address and the original bounds.
+
+[NOTE]
+====
+When performing invalid address conversion the tag never needs to be cleared because
+the earlier length violation check is a stricter condition than the representability
+check, and the capability is always unsealed before invalid address conversion.
+====
```

Comment:
I don't fully agree with this note for a couple of reasons:
1. The tag is never cleared because the instruction excepts in all cases instead of changing PCC when something goes wrong
2. The capability is not _always_ unsealed. As @arichardson pointed out, there is an exception when the immediate is not 0 because you may be jumping to an instruction that is not the entry point.

I would prefer to remove this note altogether.

Reply:
I'll just remove this so we can get the important bit landed.